core/src/banking_stage/consume_worker.rs

```diff
@@ -194,7 +194,7 @@ impl ConsumeWorkerMetrics {
                 self.error_metrics.report_and_reset(&self.id);
                 self.slot.swap(slot, Ordering::Relaxed);
             }
-        } else {
+        } else if prev_slot_id != 0 {
```

Comment:
you need to reset self.slot in here. otherwise it will still spam.

You should start up a test validator and check logs. It should not report more than once per slot.